src/components/attendee/MiniProfile/MiniProfile.tsx

```diff
@@ -20,5 +25,11 @@ export const MiniProfile: React.FC<MiniProfileProps> = ({
     return <></>;
   }
 
-  return <MiniProfileModal userId={selectedUserId} />;
+  return (
+    <div ref={setReferenceElement}>
```

Comment:
[NOTE]
Code is OK, no changes needed.

Just noting the object isn't exactly element, but has a field that is a reference to the element.
If there is a single one of these `ref` might be enough, and for multiple `divRef` or `containerRef` would be distinguishing enough.